sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -266,14 +266,34 @@ public partial class DownloadManifestOptions
     {
         public DownloadManifestOptions(string tag = null, string digest = null) { }
         public string Digest { get { throw null; } }
+        public string MediaType { get { throw null; } set { } }
         public string Tag { get { throw null; } }
     }
     public partial class DownloadManifestResult
     {
         internal DownloadManifestResult() { }
         public System.BinaryData Content { get { throw null; } }
         public string Digest { get { throw null; } }
-        public Azure.Containers.ContainerRegistry.Specialized.ArtifactManifest Manifest { get { throw null; } }
+        public string MediaType { get { throw null; } set { } }
+        public Azure.Containers.ContainerRegistry.Specialized.OciManifest AsOciManifest() { throw null; }
+    }
+    public partial class ManifestListAttributes
+    {
+        public ManifestListAttributes() { }
+        public string Digest { get { throw null; } set { } }
+        public string MediaType { get { throw null; } set { } }
+        public Azure.Containers.ContainerRegistry.Specialized.ManifestPlatform Platform { get { throw null; } set { } }
+        public long? Size { get { throw null; } set { } }
+    }
+    public partial class ManifestPlatform
+    {
+        public ManifestPlatform() { }
+        public Azure.Containers.ContainerRegistry.ArtifactArchitecture Architecture { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> Features { get { throw null; } }
+        public Azure.Containers.ContainerRegistry.ArtifactOperatingSystem OperatingSystem { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> OsFeatures { get { throw null; } }
```

Comment:
We should rename this and OsVersion to OperatingSystemFeatures and OperatingSystemVersion.